services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/outputs/_watcher.py

```diff
@@ -39,6 +41,7 @@ async def _worker_events(self) -> None:
             if event is None:
                 break
 
+            logger.debug("%s %s", _TEST_MARK, event)
```

Comment:
THOUGHT:  I do not like how you have to add something in the code to test it.
what about spying instead ``self.outputs_context.port_key_events_queue.coro_get()`` 

If you are not familiar with that concept, see https://wesmckinney.com/blog/spying-with-python-mocks/

Reply:
Let's have a look together at this. I'm also interested in seeing this at work. I do not like having test marks all over the place